do_snowDA.sh

```diff
@@ -78,17 +80,17 @@ module list
 INCDATE=${SCRIPTDIR}/incdate.sh
 
 # substringing to get yr, mon, day, hr info
-YYYY=`echo $THISDATE | cut -c1-4`
-MM=`echo $THISDATE | cut -c5-6`
-DD=`echo $THISDATE | cut -c7-8`
-HH=`echo $THISDATE | cut -c9-10`
+export YYYY=`echo $THISDATE | cut -c1-4`
```

Comment:
Why are these exported now?

Reply:
Keep consistent with the use in the submit_cycle.sh script.